pyfr/plugins/fluidforce.py

```diff
@@ -71,6 +84,8 @@ def __init__(self, intg, cfgsect, suffix):
             # Element indices and associated face normals
             eidxs = defaultdict(list)
             norms = defaultdict(list)
+            rfpts = defaultdict(list)
+            rfpts_c_norms = defaultdict(list)
```

Comment:
Not sure if pre-computing the cross products is necessarily worth it having seen both.

Reply:
Removed it.